public/human-refinement/index.js

```diff
@@ -380,6 +380,13 @@ const loadCase = async (caseId) => {
 		},
 	});
 
+	// Reset search params
+	$pdfSearchInput.value = '';
+	$pdfSearchInput.dataset.value = '';
+	$pdfSearchBar.classList.remove('active');
+	$pdfSearchInput.blur();
+	Array.from($('.searchDivPointWrap') || []).forEach((p) => p.remove());
```

Comment:
^Would make the above a re-usable function but usage in scoping makes it annoying to use elsewhere... I can add it as window.resetSearch?

Reply:
That should be covered as part of https://github.com/openlawnz/openlawnz-human-refinement-centre/issues/16